src/store/menu/menuSlice.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { RootState } from '../store';
+
+const initialState = {
+  isProfileMenuOpen: false,
+  isMobileMenuOpen: false,
+};
+
+export const menuSlice = createSlice({
```

Comment:
How many methods and variables will it contains?

My suggestion is to rename it into common or util slice and keep here everything what is shared via whole app

Reply:
add correction